api/src/main/resources/example.application.yaml

```
@@ -10,15 +10,16 @@ loginsvc:
         alg-name: "RS256"
       #Instead of generating the key in memory
       #The Below Config allows for the application to fetch keys from AWS Secrets Manager.
-      #aws-secrets-manager:
-        #secret-name: "secret"
-        #region: "region"
-        #private-key-field-name: "privateKey"
-        #public-key-field-name: "publicKey"
-        #access-exp-time: 15min
-        #refresh-exp-time: 9h
-        #poll-time: 5min
-        #alg-name: "RS256"
+      aws-secrets-manager:
+        secret-name: "secret"
+        region: "region"
+        private-key-field-name: "privateKey"
+        public-key-field-name: "publicKey"
+        access-exp-time: 15min
+        refresh-exp-time: 9h
+        poll-time: 5min
+        key-phase-out-time: 2min
```

```suggestion
        key-phase-out-time: 30min
```
(a more realistic example, but more about the pahse-out later)